practice-app/server/src/routes/chess/chess.controller.js

```diff
@@ -132,14 +162,20 @@ const ChessController = {
             });
         } catch (e) {
             console.log(e);
+            return res
+                .status(500)
+                .json({ message: "Cannot not stream the game." });
         }
     },
     getGames: async function (req, res) {
+        const user = req.auth;
+
         try {
             const games = await ChessGame.find(
-                {},
+                { user },
```

Comment:
I am asking this one for clarification. What is the type of req.auth? Is it user's id or an object with fields like id, email etc? @ersoykadir @nurlandadashov02